Cargo.toml

```diff
@@ -43,14 +43,14 @@ members = [
 futures-core = { version = "0.3", default-features = false }
 futures-sink = { version = "0.3", default-features = false }
 futures-util = { version = "0.3", default-features = false }
-tokio-util = { version = "0.3.1", features = ["codec"] }
-tokio = { version = "0.2", features = ["io-util"] }
+tokio-util = { version = "0.4.0", features = ["codec"] }
```

Comment:
Is there a reason to specify the `patch` version for `tokio-util`? _(It's not specified for most other dependencies...)_